v0/pipeline.json

```diff
@@ -45656,6 +45656,15 @@
                     "pattern" : "^<\\+input>((\\.)((executionInput\\(\\))|(allowedValues|default|regex)\\(.+?\\)))*$",
                     "minLength" : 1
                   } ]
+                },
```

Comment:
@namantalaycha10 I think we should add this to v1 as well

Reply:
Thanks for Mentioning , added the changes